File: angular/src/app/main/supplyChain/sales/shared/dtos/oedriversdto.ts
export class Oedriversdto {
    id:number;
    driverID:number;
    driverName:string;
    driverCtrlAcc:string;
    accountDesc:string;
    subAccountDesc:string;
    driverSubAccID:number;
    active: boolean;
    createdBy: string;
    createDate: string | Date;
    audtUser: string;
    audtDate: string | Date;
}
export class GetcaderDToOutput {
    OeDriversDTo: Oedriversdto
}

export class PagedResultDtocader {
    totalCount: number;
    items: GetcaderDToOutput[]
}

File: aspnet-core/src/ERP.Application.Shared/SupplyChain/Sales/OEDrivers/Dtos/OEDriversDto.cs
using System;
using Abp.Application.Services.Dto;

namespace ERP.SupplyChain.Sales.OEDrivers.Dtos
{
    public class OEDriversDto : EntityDto
    {
        public int DriverID { get; set; }

        public string DriverName { get; set; }

        public bool Active { get; set; }

        public string DriverCtrlAcc { get; set; }

        public int? DriverSubAccID { get; set; }

        public string CreatedBy { get; set; }

        public DateTime? CreateDate { get; set; }

        public string AudtUser { get; set; }

        public DateTime? AudtDate { get; set; }

    }
}